frontend/src/store/todos-context.tsx

```diff
@@ -52,12 +58,23 @@ const TodosContextProvider : React.FC = props =>
 		return setFetchedItems(todos);
 	}
 
+	function editTodoHandler(_id : number)
+	{
+		const editItem : Todo[] = fetchedItems.filter(item => item._id === _id);
+		const itemForEdit : StateTodo = editItem[0];
+
+		setItemForEdit(itemForEdit);
+
+	}
+
 	const contextValue : TodosContextModel = {
         items: todos,
         addTodo: addTodoHandler,
         removeTodo: removeTodoHandler,
         todoList,
-        fetchedItems
+        fetchedItems,
```

Comment:
fetchedItems sound like items that have been fetched... is that the case? otherwise fetchItem... in this case I dont know what you did - I have to take a look into the code